hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMajorCompaction.java

```diff
@@ -211,26 +209,9 @@ private void majorCompaction() throws Exception {
     Result result = r.get(new Get(STARTROW).addFamily(COLUMN_FAMILY_TEXT).readVersions(100));
     assertEquals(compactionThreshold, result.size());
 
-    // see if CompactionProgress is in place but null
-    for (HStore store : r.getStores()) {
-      assertNull(store.getCompactionProgress());
-    }
-
     r.flush(true);
     r.compact(true);
 
-    // see if CompactionProgress has done its thing on at least one store
-    int storeCount = 0;
-    for (HStore store : r.getStores()) {
-      CompactionProgress progress = store.getCompactionProgress();
```

Comment:
Do we have test coverage for `getProgress()` elsewhere?

Reply:
This was not the appropriate place to test progress reporting in the first place, and progress reporting was broken anyway because it was not MT safe. I do think we should add coverage for it but it should be a follow up issue that focuses on the lack of coverage (it’s always been an issue imho)

Reply:
We can file another issue for improving the test for compaction progress.